### PR TITLE
feat: Career Page update

### PR DESCRIPTION
**Pull request by shivani170**
To Update Career Page:

Replace ‘Software Engineer - Back End (Mid-level)’ with ‘SDE Back End II’
Capability Level: Mid-level
On list card: 3+ years of experience
Positions: 03
[Detail Page info](https://cdn.discordapp.com/attachments/1032870108546216007/1068404493814878218/SDE_2_Backend_JD.pdf)

Remove ‘Associate Software Engineer’

Rename ‘Mid Level DevOps Engineer’ to ‘DevOps Engineer’
Capability Level: Junior
On list card: 2+ years experience
Positions: 02

Remove ‘Product Designer’

Add ‘QA Engineer’
Capability Level: Mid-level
On list card: 3+ years experience
Positions: 03
[Detail Page info](https://cdn.discordapp.com/attachments/1032870108546216007/1068407196813119549/QA_Engineer.pdf)

Add ‘Technical Account Manager’
Capability Level: Senior
On list card: 7+ years experience
Positions: 01
[Detail Page info](https://cdn.discordapp.com/attachments/1032870108546216007/1068500360257351691/Technical_Account_Manager_JD.pdf)

//Changes in detail page:
Change first line to: Devtron was established in 2019 by industry veterans with more than 30 years of experience in building large scale companies and highly scalable software products.
In benefits: Change ‘Bangalore’ to ‘Gurugram’

Add ‘Associate Product Manager’
Capability Level: Junior
On list card: 2 years experience
Positions: 01
[Detail Page info](https://cdn.discordapp.com/attachments/1032870108546216007/1068408107019022376/Associate_Product_Manager_JD.pdf)

Rename ‘Sr. Technical Writer’ to ‘Technical Writer’
Capability Level: Mid-level
On list card: 3+ years experience
Positions: 01

### CODE REVIEW

File: public/careers.html [hunk 1]
```diff
@@ -415,12 +415,12 @@ <h3 class="section-title"><span class="regular">Job </span>Openings</h3>
         <div id="w-node-fffb6731-c2e2-2cb1-3191-222a0c69b22f-dc1f03fb" data-w-id="fffb6731-c2e2-2cb1-3191-222a0c69b22f"
           class="job-card">
           <div>
-            <h4 class="job-title"> <strong class="bold-text">Software Engineer - Back End (Mid level)</strong></h4>
+            <h4 class="job-title"> <strong class="bold-text">SDE Back End II</strong></h4>
```
Comment (sohel-rp) [inline public/careers.html:418]: Change to `SDE Backend II`

File: public/careers.html [hunk 2]
```diff
@@ -571,9 +571,63 @@ <h4 class="job-title"><strong>Lead Product Marketing Manager</strong></h4>
...
+                <div class="job-desc">2+ years of experience</div>
+              </div>
+            </div>
+          </div>
+          <a href="jobs/associate-product-manager.html" class="button w-button">Details</a>
+        </div>
         <div id="w-node-_5df7632e-7ee7-a559-7c40-aafd14fc8ad2-dc1f03fb" class="job-card">
           <div>
-            <h4 class="job-title"><strong>Sr. Technical Writer</strong></h4>
+            <h4 class="job-title"><strong> Technical Writer</strong></h4>
```
Comment (sohel-rp) [inline public/careers.html:630]: remove space before `Technical`

File: public/jobs/associate-product-manager.html [hunk 3]
```diff
@@ -0,0 +1,310 @@
+<!DOCTYPE html><!--  This site was created in Webflow. http://www.webflow.com  -->
+<!--  Last Published: Fri Feb 11 2022 14:09:25 GMT+0000 (Coordinated Universal Time)  -->
+<html data-wf-page="62055874c86e6e66811f0404" data-wf-site="61aeec2bcab0f63538b9256e">
+
+<head>
+  <meta charset="utf-8">
+  <title>Software Engineer - Backend</title>
+  <meta name="description" content="Hiring for Software Engineer - Backend">
```
Comment (sohel-rp) [inline public/jobs/associate-product-manager.html:8]: change meta info in this file to associate-product-manager specific content

File: public/jobs/qa-engineer.html [hunk 4]
```diff
@@ -0,0 +1,331 @@
+<!DOCTYPE html><!--  This site was created in Webflow. http://www.webflow.com  -->
+<!--  Last Published: Fri Feb 11 2022 14:09:25 GMT+0000 (Coordinated Universal Time)  -->
+<html data-wf-page="62055874c86e6e66811f0404" data-wf-site="61aeec2bcab0f63538b9256e">
+
+<head>
+  <meta charset="utf-8">
+  <title>Software Engineer - Backend</title>
+  <meta name="description" content="Hiring for Software Engineer - Backend">
```
Comment (sohel-rp) [inline public/jobs/qa-engineer.html:8]: Same here

File: public/jobs/sr-technical-writer.html [hunk 5]
```diff
@@ -4,8 +4,8 @@
 
 <head>
   <meta charset="utf-8">
-  <title>Sr. Technical Writer</title>
-  <meta name="description" content="Hiring for Senior Technical Writer">
+  <title> Technical Writer</title>
```
Comment (sohel-rp) [inline public/jobs/sr-technical-writer.html:7]: same here

File: public/jobs/sr-technical-writer.html [hunk 6]
```diff
@@ -114,7 +114,7 @@
   <div class="section indigo hero wf-section">
     <div class="container">
       <div data-cs="123" class="job-top">
-        <h1 class="hero-title bold white"><strong>Sr. Technical Writer</strong></h1>
+        <h1 class="hero-title bold white"><strong> Technical Writer</strong></h1>
```
Comment (sohel-rp) [inline public/jobs/sr-technical-writer.html:117]: same here

File: public/jobs/technical-account-manager.html [hunk 7]
```diff
@@ -0,0 +1,315 @@
+<!DOCTYPE html><!--  This site was created in Webflow. http://www.webflow.com  -->
+<!--  Last Published: Fri Feb 11 2022 14:09:25 GMT+0000 (Coordinated Universal Time)  -->
+<html data-wf-page="62055874c86e6e66811f0404" data-wf-site="61aeec2bcab0f63538b9256e">
+
+<head>
+  <meta charset="utf-8">
+  <title>Software Engineer - Backend</title>
+  <meta name="description" content="Hiring for Software Engineer - Backend">
```
Comment (sohel-rp) [inline public/jobs/technical-account-manager.html:8]: same here

File: public/careers.html (hunk 1 above)
Comment (shivani170) [reply public/careers.html:418]: Done

File: public/careers.html (hunk 2 above)
Comment (shivani170) [reply public/careers.html:630]: Done

File: public/jobs/qa-engineer.html (hunk 4 above)
Comment (shivani170) [reply public/jobs/qa-engineer.html:8]: Done

File: public/jobs/sr-technical-writer.html (hunk 5 above)
Comment (shivani170) [reply public/jobs/sr-technical-writer.html:7]: Done

File: public/jobs/sr-technical-writer.html (hunk 6 above)
Comment (shivani170) [reply public/jobs/sr-technical-writer.html:117]: Done

File: public/jobs/technical-account-manager.html (hunk 7 above)
Comment (shivani170) [reply public/jobs/technical-account-manager.html:8]: Done

**Review by sohel-rp — APPROVED** [review]

**Pull request merged**